### ISSUE TITLE
Broken hyperlinks

### ISSUE THREAD

**Issue by ZiggyAtl** [ocupop/comission-theme#9]
On the Museum Overview page - the hyperlink doesn’t work on the words ‘unique CoMission process’ at end of 1st para. It leads to a 404 page. I tried entering the correct page (https://comission.art/pages/how-it-works) but it says that any hyperlinks need an http: address and it won’t accept it as all the comission pages appear to be https. 

The same also applies to the ‘contact us’ words at the end of the first line of the FAQ page. I tried entering the correct link (https://comission.art/pages/contact-us) but once I save it, it adds an extra '/pages' in to the address and takes the user to https://comission.art/pages/pages/contact-us

Also neither of the hyperlinks are in a second color (blue) and underlined, so that the users know there is a hyperlink there.

**Comment by paulstroot** [comment]
I have a couple questions for you here:

1. Which page is the Museum Overview Page? Do you mean [this one](https://comission.art/pages/institution-partners)? I'm not seeing the text "unique CoMission process"

2. I'm not able to replicate any of the issues with the link redirecting to a url with the extra "/pages" in it. If you click the link in your issue above, does the same thing happen for you?

3. I will make sure they hyperlink on the FAQ page gets colored blue.

**Comment by ZiggyAtl** [comment]
@paulstroot 
I wish I had kept screen shots from last night because you must think I'm crazy. I was trying to update the hyperlink to the address it now routes to, but it wouldn't let me do it, and now it's routing to that address (!!?!). The other one I referred to is on the page https://comission.art/pages/museum-overview. I just went in to it and tried to make the same hyperlink change I tried last night (from https://comission.art/how-it-works  to  http://comission.art/pages/how-it-works) and saved everything. But when I went to the site I was still getting a 404. I went back to check the link I had updated and it hadn't remembered the saved change to the actual link however it had remembered it to the link title (so I know I'm not cracking up and did make that change). As I mentioned, there is a comment under the box which says external links need to start http:// and all our pages start http:// so I wondered if that was the issue?

**Comment by paulstroot** [comment]
It does sound confusing. But is everything working properly now @ZiggyAtl ?

**Comment by paulstroot** [comment]
Also, to clarify. All links should start with "**https**". Anything starting with "**http**" will be automatically redirected to "**https**", and if you leave out the "**http(s)**" protocol all together the link will not work.

**Comment by ZiggyAtl** [comment]
@paulstroot  yes I have now updated the one on https://comission.art/pages/museum-overview but the words 'unique CoMission process.' need the blue and underline.

**Comment by paulstroot** [comment]
Done!
